timely_beliefs/beliefs/classes.py

```diff
@@ -276,18 +277,29 @@ def add_to_session(
         """
         # Belief timing is stored as the belief horizon rather than as the belief time
         belief_records = (
-            beliefs_data_frame.convert_index_from_belief_time_to_horizon()
-            .reset_index()
-            .to_dict("records")
```

Comment:
Without this line the variable name `belief_records` no longer fits, so perhaps rename it.

Reply:
True, needs renaming